src/routes/docs/examples/Sparkline/+page.ts

```diff
@@ -0,0 +1,9 @@
+import pageSource from './+page.svelte?raw';
+
+export async function load() {
+  return {
+    meta: {
+      pageSource,
+    },
```

Comment:
I try to add `related` components when it makes sense.  Maybe add `related: ['components/Spline']` and also update `Spline/+page.js` and add `examples/Sparkline`.

Reply:
Done!